mmv1/api/resource.go

```diff
@@ -328,6 +328,8 @@ type Resource struct {
 	Compiler string
 
 	ImportPath string
+
+	ApiResourceType string `yaml:"api_resource_type"`
```

Comment:
Double-checking language used other places... we could consider calling this `api_resource_type_kind` if it's just the final piece of the name (since `api_resource_type` could be confused with the full type that includes the domain)

Reply:
Agreed, will change